doc-content/enterprise-only/weblogic/wls-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,260 @@
+[id='wls-configure-embedded-engine-proc']
+ifdef::PAM[]
+= Configuring embedded {PROCESS_ENGINE} and {DECISION_ENGINE} in {WEBLOGIC}.
+endif::[]
+
+ifdef::DM[]
+= Configuring embedded {DECISION_ENGINE} in {WEBLOGIC}
+endif::[]
+
```

Comment:
You need to add context. What is an embedded engine. So something like... An embedded engine is... It enables you to do .... If you use an embedded engine in your {PRODUCT} application, you must configure it. ....

Reply:
I will try to add a context about the embedded engine.